testing/deployer/topology/topology.go

```diff
@@ -198,6 +200,10 @@ type Cluster struct {
 	// components.
 	Enterprise bool `json:",omitempty"`
 
+	// Services is a forward declaration of V2 services. This goes in hand with
+	// the V2Services field on the Service (instance) struct.
+	Services map[ServiceID]*pbcatalog.Service `json:"omitempty"`
```

Comment:
Use of this is optional. If you elect not to use it, then v2 Services definitions are inferred from the list of instances defined here.

Reply:
this is helpful context, can you please add it to the docstring?